### PR TITLE
AC Automated Test - LRAC-9215 | master

### PR DESCRIPTION
**Pull request by kenduenwald**
https://issues.liferay.com/browse/LRAC-9215
https://issues.liferay.com/browse/LRAC-9216
https://issues.liferay.com/browse/LRAC-9217
https://issues.liferay.com/browse/LRAC-9218
https://issues.liferay.com/browse/LRAC-9219
https://issues.liferay.com/browse/LRAC-9220
https://issues.liferay.com/browse/LRAC-9221
https://issues.liferay.com/browse/LRAC-9222
https://issues.liferay.com/browse/LRAC-9223
https://issues.liferay.com/browse/LRAC-9224
https://issues.liferay.com/browse/LRAC-9225
https://issues.liferay.com/browse/LRAC-9226

### CODE REVIEW

**Comment by kenduenwald** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Reviewing...

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro [hunk 1]
```diff
@@ -359,6 +359,42 @@ definition {
...
+		var month = DateUtil.getMonthName("${dateDifferenceFromCurrentDate}");
+		var year = DateUtil.getYear("${dateDifferenceFromCurrentDate}");
+
+		static var staticDay = "${day}";
+		static var staticMonth = "${month}";
+		static var staticYear = "${year}";
+
+		return "${staticDay}";
+		return "${staticMonth}";
+		return "${staticYear}";
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro:377]: wow @kenduenwald , this is new to me! I didn't know we are able to make multiple returns on Poshi! ✨ 

cc: @ViniLopes20

Comment (kenduenwald) [reply portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro:377]: Yup, been doing this all test automation for Help Center that require date values.

**Comment by kenduenwald** [comment]
@alexcandidos, pushed one more additional correction

**Comment by alexcandidos** [comment]
hey @kenduenwald , Thanks for working on it, just to confirm, were you able to run all these tests? I mean, we can proceed without adding them to quarantine?

**Comment by kenduenwald** [comment]
hey @alexcandidos, I didn't run the tests but I structured them very similarly to tests that had already been written in that same test case. Because of that, I didn't add the quarantine label.

**Comment by alexcandidos** [comment]
Hi @kenduenwald , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**